nms/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -746,188 +897,46 @@ function AddSubscriberDetailsTable(props: AddSubscribersProps) {
-                    data-testid="activeApns"
-                    multiple
-                    value={props.value ?? []}
-                    onChange={({target}) => props.onChange(target.value)}
-                    displayEmpty={true}
-                    renderValue={selected => {
-                      if (!selected.length) {
-                        return 'Select APNs';
-                      }
-                      return selected.join(', ');
-                    }}
-                    input={
-                      <OutlinedInput
-                        className={props.value ? '' : classes.placeholder}
-                      />
-                    }>
-                    {apns.map((k: string, idx: number) => (
-                      <MenuItem key={idx} value={k}>
-                        <Checkbox
-                          checked={
-                            props.value ? props.value.indexOf(k) > -1 : false
-                          }
-                        />
-                        <ListItemText primary={k} />
-                      </MenuItem>
-                    ))}
-                  </Select>
-                </FormControl>
-              ),
-            },
-            {
-              title: 'Active Policies',
-              field: 'policies',
-              editComponent: props => (
-                <FormControl>
-                  <Select
-                    data-testid="activePolicies"
-                    multiple
-                    value={props.value ?? []}
-                    onChange={({target}) => props.onChange(target.value)}
-                    displayEmpty={true}
-                    renderValue={selected => {
-                      if (!selected.length) {
-                        return 'Select Policies';
-                      }
-                      return selected.join(', ');
-                    }}
-                    input={
-                      <OutlinedInput
-                        className={props.value ? '' : classes.placeholder}
-                      />
-                    }>
-                    {policies.map((k: string, idx: number) => (
-                      <MenuItem key={idx} value={k}>
-                        <Checkbox
-                          checked={
-                            props.value ? props.value.indexOf(k) > -1 : false
-                          }
-                        />
-                        <ListItemText primary={k} />
-                      </MenuItem>
-                    ))}
-                  </Select>
-                </FormControl>
-              ),
-            },
-          ]}
+          data={
+            subscriberAction === 'delete' && !subscribers.length
+              ? getSubscribers
+              : subscribers
+          }
+          columns={
+            !(subscriberAction === 'delete')
+              ? [
+                  {
+                    title: '',
+                    field: '',
+                    width: '5%',
```

Comment:
Use a pixel count. This should always be wide enough to display the IDs, which we can show up to 3 or 4 digits with a set pixel count. With a percentage here it might be too small or large depending on how the window is sized.